.github/workflows/publish-insiders.yml

```diff
@@ -1,9 +1,9 @@
 name: Publish For Insiders
 
 on:
-  schedule:
+#  schedule:
     # Every day at midnight
-    - cron:  '0 0 * * *'
+#    - cron:  '0 0 * * *'
```

Comment:
Do we need `workflow_dispatch`
I added that as a way for manually publishing from CI.
I.e. if you run the CI pipline manually and add the message `true` in response to the message `Publish to VSCode marketplace?` then we publish it (i.e. its still automated on CI - via a click of a button).

Or awe we now saying that we'll always pull the extension VSIX down & upload to marketplace manually?
If thats the case we should remove this dispatch

Reply:
I disagree. I'd like to dispatch from the github action